src/index.js

```diff
@@ -32,9 +33,10 @@ var Time = React.createClass({
     var machineReadable = value.format('YYYY-MM-DDTHH:mm:ssZ');
 
     if (relative || format) {
+      var titleValue = value.format(titleFormat || 'YYYY-MM-DD HH:mm');
```

Comment:
Also we don't need to format `titleValue` if `!relative` so let's move this expression down to line 39 where we check for `relative`.